python/ray/data/_internal/planner/planner.py

```diff
@@ -48,19 +48,19 @@ def _plan(self, logical_op: LogicalOperator) -> PhysicalOperator:
 
         if isinstance(logical_op, Read):
             assert not physical_children
-            physical_op = _plan_read_op(logical_op)
+            physical_op = plan_read_op(logical_op)
         elif isinstance(logical_op, InputData):
             assert not physical_children
             physical_op = _plan_input_data_op(logical_op)
         elif isinstance(logical_op, Write):
             assert len(physical_children) == 1
-            physical_op = _plan_write_op(logical_op, physical_children[0])
+            physical_op = plan_write_op(logical_op, physical_children[0])
         elif isinstance(logical_op, AbstractFrom):
             assert not physical_children
             physical_op = _plan_from_op(logical_op)
         elif isinstance(logical_op, AbstractUDFMap):
             assert len(physical_children) == 1
-            physical_op = _plan_udf_map_op(logical_op, physical_children[0])
+            physical_op = plan_udf_map_op(logical_op, physical_children[0])
         elif isinstance(logical_op, AbstractAllToAll):
             assert len(physical_children) == 1
             physical_op = _plan_all_to_all_op(logical_op, physical_children[0])
```

Comment:
let's update all `_plan_xxx` -> `plan_xxx` in a followup PR?

Reply:
I prefer to change some of them in this PR. Because this PR has standardized all function names in `plan_udf_map_op.py` , `plan_read_op.py`, and `plan_write_op.py`. I'll create another PR for the remaining.

Reply:
yeah agree, that's also I am proposing here.